tcms/testruns/templates/testruns/get.html

```diff
@@ -299,6 +299,20 @@ <h5><span class="test-executions-count"></span> Results</h5>
                                                         </li>
                                                     </template>
                                                 </div>
+                                                <strong>{% trans "History" %}: </strong>
+                                                <div>
+                                                    <ul class="history-container">
+                                                        <template id="history-entry">
+                                                            <li>
+                                                                <div>
+                                                                    <span class="history-date"></span> <span class="history-user"></span>
+                                                                </div>
+                                                                <span class="history-change-reason"></span>
+                                                            </li>
```

Comment:
These list items don't look good to me. We don't have styling for the bullet points and frankly we don't really need to use lists here.

Will a bunch of divs align to the left without any margins and look the same ? If yes then let's remove the lists.

Reply:
yes, divs allign on the left. maybe what we can do for better visual representation is indend the history container a bit to the right

Reply:
> yes, divs allign on the left. maybe what we can do for better visual representation is indend the history container a bit to the right

Nope. Things should be aligned as they are, proper rendering of the diff (not on 1 line) and syntax highlighting will make it better but this is outside the current scope.